en/sim_sih/index.md

```diff
@@ -164,6 +164,45 @@ export PX4_HOME_ALT=28.5
 make px4_sitl sihsim_quadx
 ```
 
+
+### Adding new airframes
+
+[Adding a new airframe](../dev_airframes/adding_a_new_frame.md) for use in SIH simulation is much the same as for other use cases.
+You still need to configure your vehicle type and [geometry](../config/actuators.md) (`CA_` parameters) and start any other defaults for that specific vehicle.
+
+The specific differences for SIH simulation airframes are listed in the sections below.
+
+For all variants of SIH:
+
+  - Set all the [Simulation In Hardware](../advanced_config/parameter_reference.md#simulation-in-hardware) parameters (prefixed with `SIH_`) in order to configure the physical model of the vehicle.
+  - `param set-default SYS_HITL 2` to enable SIH on the next boot.
```

Comment:
As I understand https://github.com/PX4/PX4-user_guide/pull/3538#issuecomment-2606578011 this is true for both SIH on FC and Simulator. It makes sense to state what the param does so that people know they don't need to start these sensors.

```suggestion
  - `param set-default SYS_HITL 2` to enable SIH on the next boot.
  
     This disables input from real sensors and enables simulated sensor data for [GPS](../advanced_config/parameter_reference.md#simulation-in-SENS_EN_GPSSIM), [barometer](../advanced_config/parameter_reference.md#simulation-in-SENS_EN_BAROSIM ), [magnetometer](../advanced_config/parameter_reference.md#simulation-in-SENS_EN_MAGSIM ) and [Airspeed](../advanced_config/parameter_reference.md#simulation-in-SENS_EN_ARSPDSIM) (used in fixed-wing flight).
```

Reply:
This is indeed only true for SIH-as-SITL, not for SIH on FC. I did not completely understand that when writing https://github.com/PX4/PX4-user_guide/pull/3538#issuecomment-2606578011, sorry for any confusion.